pest/src/iterators/pair.rs

```diff
@@ -123,6 +123,35 @@ impl<'i, R: RuleType> Pair<'i, R> {
     /// ```
     #[inline]
     pub fn into_span(self) -> Span<'i> {
+        // into_span was here first, so it is left in for backwards compatibility
+        self.as_span()
+    }
+
+    /// Returns the `Span` defined by the `Pair`, **without** consuming it.
```

Comment:
I would keep the documentation the same for both. The only difference is that the deprecated one should point here.